core/src/components/refresher/test/a11y/refresher.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import AxeBuilder from '@axe-core/playwright';
+import { expect } from '@playwright/test';
+import { configs, test, dragElementBy } from '@utils/test/playwright';
+
+configs({ directions: ['ltr'], modes: ['md'], themes: ['light', 'dark'] }).forEach(({ config, title }) => {
+  test.describe(title('refresher: a11y for ion-color()'), () => {
+    test('should not have accessibility violations', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-content>
+          <ion-refresher class="refresher-native" slot="fixed">
+            <ion-refresher-content></ion-refresher-content>
+          </ion-refresher>
+        </ion-content>
+      `,
+        config
+      );
+
+      const target = page.locator('body');
+
+      await page.waitForSelector('ion-refresher.hydrated', { state: 'attached' });
```

Comment:
Why is this needed?

Reply:
I was following the [structure](https://github.com/ionic-team/ionic-framework/blob/888a30fba5790c2b21b669aa6a58937914d5336e/core/src/components/refresher/test/test.utils.ts#L17) from `pullToRefresh()` but it still works without it. I'll remove it.